packages/react-components/src/molecules/__tests__/LabeledDateField.test.tsx

```diff
@@ -1,31 +1,57 @@
 import { formatISO } from 'date-fns';
-import { render, fireEvent } from '@testing-library/react';
+import { render, fireEvent, screen } from '@testing-library/react';
 
 import LabeledDateField from '../LabeledDateField';
 
 it('renders a labeled date field, with the stringified value', () => {
-  const { getByLabelText } = render(
+  render(
     <LabeledDateField title="Date" value={new Date('2020-01-01T12:34:56')} />,
   );
-  expect(getByLabelText('Date')).toHaveAttribute('type', 'date');
-  expect(getByLabelText('Date')).toHaveValue('2020-01-01');
+  expect(screen.getByLabelText('Date')).toHaveAttribute('type', 'date');
+  expect(screen.getByLabelText('Date')).toHaveValue('2020-01-01');
 });
 
 it('renders an empty date field', () => {
-  const { getByLabelText } = render(<LabeledDateField title="Date" />);
-  expect(getByLabelText('Date')).toHaveValue('');
+  render(<LabeledDateField title="Date" />);
+  expect(screen.getByLabelText('Date')).toHaveValue('');
+});
+
+it('renders a labeled date field, passing through props', () => {
+  render(
+    <LabeledDateField
+      title="Publish Date"
+      subtitle="('required')"
+      description={'Please enter a date before today'}
+    />,
+  );
+  expect(screen.getByText(/publish date/i)).toBeVisible();
+  expect(screen.getByText(/required/i)).toBeVisible();
+  expect(screen.getByText(/please enter a date before today/i)).toBeVisible();
+});
+
+it('supports max date attribute', () => {
+  render(
+    <LabeledDateField
+      title="Date"
+      value={new Date('2034-02-01')}
+      max={new Date()}
```

Comment:
In 12 years this test will fail 😅 
How about specifying the year?

Reply:
I updated the tests. If we set the max date to tomorrow and the current value to today, it'll never fail